interfaces/security.go

```diff
@@ -23,6 +23,25 @@ import (
 	"fmt"
 )
 
+// LauncherNameForApp returns the launcher name for a given application.
+//
+// In general, the launcher has the form: "$snap.$app". When both snap name and
+// app name are the same then the tag is simplified to just "$snap".
+func LauncherNameForApp(snapName, appName string) string {
```

Comment:
For some reason "Launcher" reminds me off "ubuntu-core-launcher" or "snap-launcher". This is the name of the executable command, right? Maybe `BinaryNameForApp` or `ExecutableNameForApp` or `GeneratedExecutableNameForApp` or somesuch? Just to add more clarity?